tests/main.js

```diff
@@ -2758,3 +2780,109 @@ test('listFunctionsFiles does not include wrong arch functions and warns', async
 
   console.warn.restore()
 })
+
+testMany(
```

Comment:
Should we assert that any configuration properties passed via the existing `config` object are kept even if there's a function-specific configuration file that doesn't overwrite it?

Reply:
Could be nice to also assert that the property from the function-specific config takes priority.

Reply:
Added the tests